src/screens/ProfileScreen/index.js

```diff
@@ -687,7 +701,7 @@ const ProfileScreen = ({route}) => {
         onOpenImageGalery={() => onOpenImageGalery()}
         onOpenCamera={() => onOpenCamera()}
         handleRemoveImageProfile={() => handleRemoveImageProfile()}
-        isLoadingUpdateImageGalery={isLoadingUpdateImageGalery}
+        isLoadingUpdateImageGallery={isLoadingUpdateImageGallery}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There's a typo in the prop name. It should be `isLoadingUpdateImageGallery` instead of `isLoadingUpdateImageGalery`.
```diff
-         isLoadingUpdateImageGalery={isLoadingUpdateImageGalery}
+         isLoadingUpdateImageGallery={isLoadingUpdateImageGallery}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->